pkg/controller/certificates/rootcacertpublisher/publisher.go

```diff
@@ -98,8 +98,9 @@ func (c *Publisher) Run(ctx context.Context, workers int) {
 	defer utilruntime.HandleCrash()
 	defer c.queue.ShutDown()
 
-	klog.Infof("Starting root CA certificate configmap publisher")
-	defer klog.Infof("Shutting down root CA certificate configmap publisher")
+	logger := klog.FromContext(ctx)
+	logger.Info("Starting root CA publisher")
+	defer logger.Info("Shutting down root CA publisher")
```

Comment:
IMO this should not deviate much from the controller name

Reply:
So I  can change it to:
```
	logger.Info("Starting root-ca-cert-publisher-controller")
	defer logger.Info("Shutting down root-ca-cert-publisher-controller")
```
Is that OK?

Reply:
I think the second one suits this better - better to not change the things too much

Reply:
Okay. Updated.